test/e2e/mcd_test.go

```diff
@@ -210,8 +210,8 @@ func TestKernelType(t *testing.T) {
 
 	assert.Equal(t, infraNode.Annotations[constants.CurrentMachineConfigAnnotationKey], oldInfraRenderedConfig)
 	assert.Equal(t, infraNode.Annotations[constants.MachineConfigDaemonStateAnnotationKey], constants.MachineConfigDaemonStateDone)
-	kernelInfo = execCmdOnNode(t, cs, infraNode, "uname", "-a")
-	if strings.Contains(kernelInfo, "PREEMPT RT") {
+	kernelInfo = execCmdOnNode(t, cs, infraNode, "chroot", "/rootfs", "rpm", "-qa", "kernel-rt-core")
+	if strings.Contains(kernelInfo, "kernel-rt-core") {
```

Comment:
```suggestion
	if strings.hasPrefix(kernelInfo, "kernel-rt-core") {
```

Reply:
Thanks for the feedback! let's fix this in a followup PR to unblock tests and pending prs.